packages/client/bin/cli.ts

```diff
@@ -828,6 +890,8 @@ async function run() {
       chain: chainName,
       mergeForkIdPostMerge: args.mergeForkIdPostMerge,
     })
+    //@ts-ignore
+    common.customCrypto = cryptoFunctions
```

Comment:
Nit: should likely add this `customCrypto` to the `fromGethGenesis` constructor also?

Reply:
Good point.  Maybe the next PR.